doctrine/pt.html

```diff
@@ -0,0 +1,289 @@
+      <div class="heading__headline common-headline">
+        <h1>A Doutrina Rails.</h1>
+      </div>
+      <div class="heading__content heading__content--has-headline common-content">
+        <dl>
+          <dt>David Heinemeier Hansson</dt>
+        </dl>
+      </div>
+      <div class="language">
+        <input class="language__checkbox" id="language__checkbox" type="checkbox" role="button" />
+        <label class="language__toggle" for="language__checkbox">
+          <span>Português</span>
+        </label>
+        <div class="language__options">
+          <ul>
+            <li><a href="/doctrine/de"><span>Deutsch</span></a></li>
+            <li><a href="/doctrine/es"><span>Español</span></a></li>
+            <li><a href="/doctrine/fr"><span>Français</span></a></li>
+            <li><a href="/doctrine/ru"><span>Russian</span></a></li>
+            <li><a href="/doctrine/zh_cn"><span>简体中文</span></a></li>
+            <li><a href="/doctrine/zh_tw"><span>繁體中文</span></a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="text common-background--grey common-padding--bottom common-padding--top-small common-shape--bottom-white-down-right">
+  <div class="container">
+    <div class="text__body">
+      <div class="text__content common-content">
+        <p>A ascensão fenomenal do Ruby on Rails à proeminência deveu-se muito à tecnologia inovadora e ao timing. Mas as vantagens tecnológicas se desgastam com o tempo, e um bom timing não sustenta movimentos sozinhos a longo prazo. Portanto, é necessária uma explicação mais ampla de como o Rails continuou não apenas a se manter relevante, mas a aumentar seu impacto e comunidade. Proponho que o facilitador duradouro tenha sido e continue sendo sua doutrina controversa.</p>
+        <p>Essa doutrina evoluiu na última década, mas a maioria de seus pilares mais fortes também são os fundadores. Não faço nenhuma reivindicação à originalidade fundamental dessas ideias. A principal realização do Rails foi unir e cultivar uma tribo forte em torno de um amplo conjunto de pensamentos heréticos sobre a natureza da programação e dos programadores.</p>
+        <p>Com toda essa introdução, seguem os nove pilares mais importantes da Doutrina Rails, conforme percebido por este que vos fala:</p>
+        <ol>
+          <li><a href="#optimize-for-programmer-happiness">Otimize para a felicidade do programador</a></li>
+          <li><a href="#convention-over-configuration">Convenção sobre Configuração</a></li>
+          <li><a href="#omakase">O menu é omakase</a></li>
+          <li><a href="#no-one-paradigm">Nenhum paradigma único</a></li>
+          <li><a href="#beautiful-code">Exalte o código bonito</a></li>
+          <li><a href="#provide-sharp-knives">Forneça facas afiadas</a></li>
+          <li><a href="#integrated-systems">Valorize sistemas integrados</a></li>
+          <li><a href="#progress-over-stability">Progresso sobre estabilidade</a></li>
```

Comment:
```suggestion
          <li><a href="#progress-over-stability">Progresso antes de estabilidade</a></li>
```
Pelas mesmas razões do "Convention over Configuration"